File: official/guides/getting-started/java/create_to_address.java
package getting_started;

import java.util.HashMap;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;

public class CreateToAddress {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        Map<String, Object> toAddressMap = new HashMap<String, Object>();
        addressMap.put("name", "George Costanza");
        addressMap.put("company", "Vandelay Industries");
        addressMap.put("street1", "1 E 161st St.");
        addressMap.put("city", "Bronx");
        addressMap.put("state", "NY");
        addressMap.put("zip", "10451");

        Address.create(toAddressMap);
    }
}

File: official/guides/getting-started/golang/create_to_address.go
toAddress, err := client.CreateAddress(
  &easypost.Address{
    Name:   "George Costanza",
    Company: "Vandelay Industries",
    Street1: "1 E 161st St",
    City: "Bronx",
    State: "NY",
    Zip: "10451",
  },
  &easypost.CreateAddressOptions{}
)
